ereolen_support/ereolen_support.helper.inc

```diff
@@ -17,10 +17,9 @@
  *
  * @throws \Exception
  */
-function _ereolen_support_search_jira_user($user, $form_state) {
-  $form_email = $form_state['values']['ereolen_support_email'];
+function _ereolen_support_search_jira_user($user, $form_state, $email) {
   $requestParts = _ereolen_support_get_request_parts('GET');
-  $result = drupal_http_request($requestParts['jira_path'] . '/user/search?includeInactive=true&username=' . $form_email, $requestParts['options']);
+  $result = drupal_http_request($requestParts['jira_path'] . '/user/search?includeInactive=true&username=' . $email, $requestParts['options']);
```

Comment:
You may want to use https://www.php.net/manual/en/function.urlencode.php here. Or https://www.php.net/manual/en/function.http-build-query.php.

Reply:
Noted!